torchreid/models/inceptionv4_pytcv.py

```diff
@@ -615,9 +612,8 @@ def __init__(self,
         self.bn_frozen = bn_frozen
         self.pooling_type = pooling_type
         self.loss = loss
-        self.feature_dim = feature_dim
         self.self_challenging_cfg = self_challenging_cfg
-        self.lr_finder = lr_finder
+        self.num_features = 1536
```

Comment:
Why it is hardcoded now?

Reply:
because I couldn't find the way how to obtain this number from the model. This number was also hardcoded in one of the Inception blocks